nifi-nar-bundles/nifi-shopify-bundle/nifi-shopify-processors/src/main/java/org/apache/nifi/processors/shopify/GetShopify.java

```diff
@@ -327,6 +327,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
                     if (objectCountHolder.get() > 0) {
                         flowFile = session.putAttribute(flowFile, CoreAttributes.MIME_TYPE.key(), "application/json");
                         session.transfer(flowFile, REL_SUCCESS);
+                        session.getProvenanceReporter().receive(flowFile, shopifyRestService.getBaseUrl());
```

Comment:
`baseUrl` only contains the value of the Store Domain property (a hostname basically). (the variable name could be improved too)
The real base URL should be logged in the provenance event.
The existing `getBaseUri()` method returns a builder containing that base URL. It could be exposed to the processor class (preferably returning a string instead of the builder).